python/lsst/ts/wep/task/calcZernikesNeuralTask.py

```diff
@@ -0,0 +1,1775 @@
+
+        # Create the table
+        donut_table = QTable(donut_data)
+
+        # Add units matching standard format
+        donut_table["coord_ra"].unit = u.rad  # Use radians like standard
+        donut_table["coord_dec"].unit = u.rad  # Use radians like standard
+        # Note: centroid_x and centroid_y are kept as plain floats to avoid
+        # Point2D constructor issues
+        donut_table["fx"].unit = u.deg
+        donut_table["fy"].unit = u.deg
+
+        # Add comprehensive visit_info metadata matching standard format
+        try:
+            visitInfo = exposure.visitInfo
+
+            # Get visit ID
+            try:
+                exp_id = exposure.getInfo().getId()
+                if hasattr(exp_id, 'getVisitId'):
+                    visit_id = exp_id.getVisitId()
+                else:
+                    visit_id = exp_id
+            except (AttributeError, NameError):
+                visit_id = visitInfo.id
+
+            # Create visit_info matching the standard structure from
+            # addVisitInfoToCatTable
+            catVisitInfo = {}
+
+            # Boresight coordinates
+            visitRaDec = visitInfo.boresightRaDec
+            catVisitInfo["boresight_ra"] = visitRaDec.getRa().asDegrees() * u.deg
+            catVisitInfo["boresight_dec"] = visitRaDec.getDec().asDegrees() * u.deg
+
+            # Boresight altitude/azimuth
+            visitAzAlt = visitInfo.boresightAzAlt
+            catVisitInfo["boresight_alt"] = visitAzAlt.getLatitude().asDegrees() * u.deg
+            catVisitInfo["boresight_az"] = visitAzAlt.getLongitude().asDegrees() * u.deg
+
+            # Rotation angles
+            catVisitInfo["boresight_rot_angle"] = visitInfo.boresightRotAngle.asDegrees() * u.deg
+            catVisitInfo["rot_type_name"] = visitInfo.rotType.name
+            catVisitInfo["rot_type_value"] = visitInfo.rotType.value
+            catVisitInfo["boresight_par_angle"] = visitInfo.boresightParAngle.asDegrees() * u.deg
+
+            # Focus and timing
+            catVisitInfo["focus_z"] = visitInfo.focusZ * u.mm
+            catVisitInfo["mjd"] = visitInfo.date.toAstropy().tai.mjd
+            catVisitInfo["visit_id"] = visit_id
+            catVisitInfo["instrument_label"] = visitInfo.instrumentLabel
+
+            # Observatory info
+            catVisitInfo["observatory_elevation"] = visitInfo.observatory.getElevation() * u.m
+            catVisitInfo["observatory_latitude"] = visitInfo.observatory.getLatitude().asDegrees() * u.deg
+            catVisitInfo["observatory_longitude"] = visitInfo.observatory.getLongitude().asDegrees() * u.deg
+            catVisitInfo["ERA"] = visitInfo.era.asDegrees() * u.deg
+            catVisitInfo["exposure_time"] = visitInfo.exposureTime * u.s
+
+            # Estimate donut diameter using DonutSizeCorrelator
+            try:
+                from lsst.ts.wep.donutSizeCorrelator import DonutSizeCorrelator
```

Comment:
You're inside `ts_wep` here, so you should be able to move that import to the top of the file.